src/neptune/metadata_containers/metadata_container.py

```diff
@@ -193,10 +193,51 @@ def __delitem__(self, path) -> None:
 
     @ensure_not_stopped
     def assign(self, value, *, wait: bool = False) -> None:
+        """Assign values to multiple fields from a dictionary.
+        You can use this method to quickly log all run's parameters.
```

Comment:
@normandy7 these are docstrings copied from `Run` but now all Neptune objects (`Model`, `ModelVersion` & `Project`) will inherit the same docstrings for all universal methods https://docs.neptune.ai/api/universal/ . We should probably tweak it to cover something much more "general" and change current Run-specific wording or add examples for all objects.